### PR TITLE
RELATED: RAIL-1815 add missing docs

### PR DESCRIPTION
**Pull request by no23reason**
* make Angular docs up-to-date (tested using a brand new Angular 10 app)
* update supported technologies section
* add docs about the available auth providers

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/459/ : SUCCESS in 1m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/266/ : SUCCESS in 5s

File: docs/02_start__connecting_backend.md [hunk 1]
```diff
@@ -44,4 +46,36 @@ The following procedure implements the initial setup using the providers to pass
...
+
+* `ContextDeferredAuthProvider`
+* `FixedLoginAndPasswordAuthProvider`
+
+You can also create a custom provider.
+
+### ContextDeferredAuthProvider
+
+The `ContextDeferredAuthProvider` assumes that the session is already authenticated to the GoodData platform (either by a login or SSO).
+This is useful in cases where you embed GoodData.UI elements into your app.
```
Comment (lupko) [inline docs/02_start__connecting_backend.md:59]: can you please detail out how to use bear's SSO functions when the analytical backend is using ContextDeferredAuthProvider?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/460/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/267/ : SUCCESS in 4s

**Review by lupko — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/461/ : SUCCESS in 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/33/ : FAILURE in 3m 14s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/125/ : SUCCESS in 12s

**Pull request merged**